google/cloud/bigtable/row_set.py

```diff
@@ -109,6 +109,24 @@ def add_row_range_from_keys(
         row_range = RowRange(start_key, end_key, start_inclusive, end_inclusive)
         self.row_ranges.append(row_range)
 
+    def add_row_range_by_prefix_from_keys(self, row_key_prefix):
```

Comment:
This name seems a bit verbose. Could we name this something a bit more concise?
`add_range_with_prefix` for instance?